solution-base/kubedb/kubedb-catalog-redis.yaml

```diff
@@ -0,0 +1,219 @@
+---
```

Comment:
same here

Reply:
I agree since we have never released yet, but once we have more than 1 release, previous catalogs should be available to avoid hypothetical issues.